packages/api-client/src/views/Settings/SettingsGeneral.vue

```diff
@@ -13,36 +18,38 @@ const {
   defaultProxyUrl,
 } = useWorkspace()
 
-const themeIds: ThemeId[] = [
+const themeIds: Exclude<ThemeId, IntegrationThemeId>[] = [
```

Comment:
Instead of this you can use satisfies for a bit cleaner look
```ts
const themeIds = [
...
] satisfies ThemeId[]
```

Reply:
We could probably just add a type in `@scalar/themes` with the preset ids tbh